src/OSFramework/Maps/Helper/Constants.ts

```diff
@@ -106,12 +106,16 @@ namespace OSFramework.Maps.Helper.Constants {
     /** URL for GoogleMapsApis  */
     /************************** */
     export const googleMapsApiURL = 'https://maps.googleapis.com/maps/api';
-    /** URL for GoogleMaps API to get coordinates by Address/Location */
-    export const googleMapsApiGeocode = `${googleMapsApiURL}/geocode/json`;
     /** URL for GoogleMaps API to make use of the Google Map */
     export const googleMapsApiMap = `${googleMapsApiURL}/js`;
     /** URL for GoogleMaps API to make use of the Google StaticMap */
     export const googleMapsApiStaticMap = `${googleMapsApiURL}/staticmap`;
+    /** Version of the google maps to be loaded. */
```

Comment:
small detail but in the comment we should have `Google Maps` instead of having it in lowercase